server.py

```diff
@@ -550,6 +554,42 @@ def get(self):
 
         return return_dict
 
+class PendingMergesResource(restful.Resource):
+    def get(self):
+        pass
+
+    def put(self, region):
+        print "entering merge_put"
+        dao = Dao(region, mongo_client=mongo_client)
+        requesting_user = get_user_from_access_token(request.headers, dao)
+        if not is_user_admin_for_region(requesting_user, region): #wow, such auth
```

Comment:
so we're letting admins for any region suggest merges? if so, I need to delete one of my test cases. just double-checking before i do that